tools/pipeline-generator/Azure.Sdk.Tools.PipelineGenerator/Conventions/WeeklyIntegrationTestingPipelineConvention.cs

```diff
@@ -23,10 +23,12 @@ protected override Schedule CreateScheduleFromDefinition(BuildDefinition definit
             var bucket = definition.Id % TotalBuckets;
             var startHours = bucket / BucketsPerHour;
             var startMinutes = bucket % BucketsPerHour;
+            var daysToBuild = new ScheduleDays[]{ ScheduleDays.Saturday, ScheduleDays.Sunday };
+            var dayBucket = definition.Id % daysToBuild.Length;
```

Comment:
Hopefully we don't end up with a lot of even or odd id's which would cause more on one side or the other.

Reply:
Currently there are 30 odd and 32 even definition IDs. If it gets super unbalanced we can figure something out or manually re-balance.

Reply:
Though even if we have 1.5x as many on one day over another, there are still 96 15-minute buckets they get spread across, so I don't think the impact will be too bad compared to our normal weekday load.